src/main/main.js

```diff
@@ -112,7 +118,8 @@ class Main {
         headers: { 'Content-Type': 'application/json' }
       });
 
-      const url = `https://idyll.pub/post/${alias}/`;
+      // const url = `https://idyll.pub/post/${alias}/`;
```

Comment:
I like the `IDYLL_PUB_URL` constant :) As such, looks like we can remove this comment :]